testapi.pm

```diff
@@ -1261,10 +1265,13 @@ object (in this example) - so it will ensure the console is active,
 but to setup the root shell on this console, the distribution needs
 to run test code.
 
+After the console selection the distribution callback
+C<$distri->console_selected> is called with C<%args>.
+
 =cut
 
 sub select_console {
-    my ($testapi_console) = @_;
+    my ($testapi_console, %args) = @_;
```

Comment:
That's a feature (not a misfeature), but i agree that `@args` would seem appropriate here.